src/api/app/models/staging/workflow.rb

```diff
@@ -60,4 +61,21 @@ def create_staging_projects
       staging_project.store
     end
   end
+
+  def update_staging_projects_managers_group
+    return unless changes[:managers_group_id]
+
+    old_managers_group = Group.find(changes[:managers_group_id].first)
+    new_managers_group = managers_group
+
+    staging_projects.each do |staging_project|
+      staging_project.unassign_managers_group(old_managers_group)
+      staging_project.assign_managers_group(new_managers_group)
+      staging_project.store
+    end
+
+    # FIXME: This assignation is need because after store a staging_project
+    # the object is reloaded and we lost the changes.
```

Comment:
Wondering why it is lost because when you reload it should be there, no?

Reply:
Tried in the console. See the managers_group_id value on top and on the bottom, it just changes back after a store of the first staging project:
![image](https://user-images.githubusercontent.com/11314634/48708929-423b3180-ec04-11e8-84d3-fc5b7c8a641b.png)

Looks like the store of the `StagingProject` reloads the staging workflow and then the old managers groups is recovered...